manifests/operatorconfiguration.crd.yaml

```diff
@@ -334,6 +334,8 @@ spec:
                   logical_backup_schedule:
                     type: string
                     pattern: '^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$'
+                  logical_backup_job_prefix:
+                    type: string
```

Comment:
can you stick to alphabetical sorting here and in the sample configuration file? Maybe docs, too. Makes things easier to find.

Reply:
We did not follow alphabetical order in the logical backup settings at all. I restored the order in a separate commit.